packages/utils/src/lib/transform.ts

```diff
@@ -37,14 +38,14 @@ export function deepClone<T>(obj: T): T {
   return obj == null || typeof obj !== 'object' ? obj : structuredClone(obj);
 }
 
-export function factorOf<T>(items: T[], filterFn: (i: T) => boolean): number {
-  const itemCount = items.length;
-  // early exit for empty items
+export function factorOf<T>(rows: T[], filterFn: (i: T) => boolean): number {
+  const itemCount = rows.length;
+  // early exit for empty rows
   if (!itemCount) {
     return 1;
   }
-  const filterCount = items.filter(filterFn).length;
-  // if no items result from the filter fn we forward return 1 as factor
+  const filterCount = rows.filter(filterFn).length;
+  // if no rows result from the filter fn we forward return 1 as factor
   return filterCount === 0 ? 1 : (itemCount - filterCount) / itemCount;
```

Comment:
👨‍💻 Readability: The "items" are mostly issues not rows. Even so, the name of the variable should stay generic for reusability. Please revert.

Reply:
reverted. But imho nitpick and should be suggested as optional feedback.